docs/conf/prague/2020/news/prg-covid-19-update-01.rst

```diff
@@ -0,0 +1,40 @@
+:template: {{year}}/generic.html
+
+.. post:: March 16, 2020
+   :tags: prague-2020, covid-19
+
+Prague Conference - COVID-19 Update 01
+======================================
+
+The most recent COVID-19 update is:
+
+.. postlist:: 1
+  :date: %A, %B %d, %Y
+  :format: {title} - {date}
+  :list-style: none
+  :tags: covid-19
+
+
+Dear documentarians in Europe and beyond,
+
+By now you should have gotten an announcement from us about ticket sales and Call for Proposals for the Prague conference. 
+However, as you're all aware, the COVID-19 virus has been causing tremendous shifts and upheavals globally, and we are following developments as they might impact our community.
+
+Last week we announced with a heavy heart that we must :doc:`postpone our Portland conference </conf/portland/2020/news/covid-19-update-01>` from May to August (exact dates to be announced soon). 
+Since the Prague conference is scheduled for September, we'd like to maintain our cautious optimism that we can hold it as planned. 
+
+**As of today, March 16, Write the Docs Prague 2020 is scheduled to go ahead**.
+
+Despite the severe measures taken by the Czech authorities during the past week, these are short-term efforts to prevent the exponential spreading of the virus, and the restrictions are due to be lifted by mid-April.
```

Comment:
A nice-to-have suggestion: could we link here to something about the Czech authorities' measures and the expectation that they'll be lifted in April?

Reply:
Good thinking! Added now :)

Reply:
I mean added whatever I could find, unfortunately the detailed documents are only in Czech, but the ministry of health has a dedicated domain so that should be sufficient for now I hope!

Reply:
I don't see any links here? Perhaps you forgot to push it up?